### ISSUE TITLE
When curl fail, the status still show OK

### ISSUE THREAD

**Issue by keyboardfann** [mathewmeconry/check_couchbase#1]
When curl a non-exist cluster , the result still show ok.

```
 ./check_couchbase1.sh -cc -c 0 -u xxxxxxxx -p xxxxxxxxx -P 8091 -h 10.62.247.92
OK - 0 of 0 healthy|FAILED=0;0;0;; WARMUP=0;;;; HEALTHY=0;;;;
```

**Comment by keyboardfann** [comment]
Please check the merge request if you are happy to merge it.
#2

**Comment by mathewmeconry** [comment]
Thanks for the pull.
I merged it and released it under v1.5
